nmostesting/suites/IS0402Test.py

```diff
@@ -2445,7 +2487,8 @@ def post_resource(self, test, type, data=None, reg_url=None, codes=None, fail=Te
                                              "current protocol: Location: {}".format(location)))
 
         # Currently can only validate schema for the API version under test
-        if reg_url == self.reg_url:
+        # TODO: Schemas for 401/403 responses don't exist in the RAML at present
```

Comment:
I believe we we sided on using the generic `error.json`, Would that help close this TODO?

Reply:
I think so!

Reply:
Ah, not the error schema, but those specific error responses for each endpoint... which isn't mentioned in the RAML  but is covered by https://github.com/AMWA-TV/nmos-discovery-registration/blob/v1.3.x/docs/2.0.%20APIs.md#error-codes--responses:
> As explicit handling of every possible HTTP response code is not expected, clients must instead implement more generic handling for ranges of response codes (1xx, 2xx, 3xx, 4xx and 5xx). For HTTP codes 400 and upwards, a JSON format response MUST be returned as follows:

Reply:
How about we actually implement that part of the spec, by modifying `get_schema` to return the `error.json` schema for all 4xx responses that aren't covered by an explicit `responses` entry in the RAML?